.github/workflows/lint-yaml.yml

```
@@ -0,0 +1,20 @@
+name: Lint yaml
+on:
+  pull_request:
+    branches:
+      - main
+      - feat/**
+  workflow_dispatch:
```

```suggestion
  workflow_dispatch: { }
```

Maybe to indicate that it is intentionally empty?

---

I don't have strong feelings here. We can either make it mandatory to indicate explicitly that it's empty and the type.
Let's keep that for an eventual follow-up when team members what this defined. The config option is there now, we can extend it as needed.